common/djangoapps/external_auth/views.py

```diff
@@ -263,75 +259,6 @@ def _flatten_to_ascii(txt):
         return unicode(unicodedata.normalize('NFKD', txt).encode('ASCII', 'ignore'))
 
 
-@ensure_csrf_cookie
-def _signup(request, eamap, retfun=None):
```

Comment:
@rlucioni The function which is calling ```_signup``` is also being called from other places, I am not sure about removing all those functions and what if those function's are being used by MIT or Stanford?